common/spec/specfiles.go

```diff
@@ -194,6 +197,9 @@ func ValidateSpec(files []File, isTargetMandatory, isSearchBasedSpec bool) error
 		isBypassArchiveInspection, _ := file.IsBypassArchiveInspection(false)
 		isTransitive, _ := file.IsTransitive(false)
 
+		if isPathMapping && !isAql {
+			return errorutils.CheckErrorf("mappings supported only with Aql")
```

Comment:
mappings supported only with Aql
-->
path mapping is supported only with Aql

Reply:
Are you sure because "mappings" is the interface to the user, they will use "mappings" in their spec.json and I saw in the code you treat it as PathMappings so I kept the same conventions, and thought that the error should show the user what he needs to fix.

Reply:
OK. NP.